index.html

```diff
@@ -4,15 +4,189 @@
-      <h1 class="title">this is the header</h1>
+    <header class="toolbar">
+      <menu class="menu_items">
+        <p class="close_menu_icon">X</p>
+        <ul>
+          <li><a href="./index.html" tabindex="1">Home</a></li>
+          <li><a href="./about.html" tabindex="2">About</a></li>
+          <li><a href="#main_program" tabindex="3">Program</a></li>
+          <li><a href="#speakers" tabindex="4">Speakers</a></li>
+          <li><a href="#sponsors" tabindex="5"> Sponsors</a></li>
+        </ul>
+      </menu>
+      <nav class="menu_icon">
+        <i class="fa-solid fa-bars" id="menu_icon"></i>
+      </nav>
+      <div class="nav_items">
+        <div class="toolbar_top">
+          <ul class="socials">
+            <li class="icons icons1">
+              <a href="https://twitter.com/EzzyWealth"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="icons icons2">
+              <a href="https://www.linkedin.com/in/ezekiel-udiomuno-b14539150/">
+                <i class="fa-brands fa-linkedin-in"></i
+              ></a>
+            </li>
+
+            <li class="icons icons4">
+              <a href="https://github.com/Ezzywealth"
+                ><i class="fa-brands fa-github"></i
+              ></a>
+            </li>
+          </ul>
+          <ul>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+          </ul>
+        </div>
+        <div class="toolbar_bottom">
+          <a href="./index.html"
+            ><img src="./images/javascript-39424.png" alt="logo" class="logo"
+          /></a>
+
+          <nav>
+            <a href="#" tabindex="1">Home</a>
+            <a href="./about.html" tabindex="2">About</a>
+            <a href="#main_program" tabindex="3">Program</a>
+            <a href="#speakers" tabindex="4">Speakers</a>
+            <a href="#sponsors" tabindex="5"> Sponsors</a>
+            <h3 class="js_camp">JS Campaign</h3>
+          </nav>
+        </div>
+      </div>
     </header>
     <main>
-      <section>this is the headline section</section>
+      <section class="headline_section">
+        <article class="headline_item">
+          <h1 class="headline_header">
+            #1 Javascript <br />
+            Conference On the Planet
+          </h1>
+          <p class="headline_description">
+            Join us, in the first Nigerian edition, of the most prestigious
+            JavaScript conference. <br />
+            2days of international speakers, community, learning and
+            connections.
+          </p>
+          <h4 class="headline_time">2023.02.25(Wed) ~ 27(Fri)</h4>
+          <h4 class="headline_venue">@ Eko Hotels, Lagos, Nigeria</h4>
+        </article>
+      </section>
+
+      <section id="main_program">
+        <article class="program">
+          <h1 class="program_header">Main Program</h1>
+          <span class="program_header_underline"></span>
```

Comment:
- [ ] For semantics reasons, kindly make sure your home page has only **one** `h1` tag which is the **page title**. For other titles, you can use other `hx(h2, h3,...)` tags.